modules/data/user.py

```diff
@@ -44,7 +42,7 @@ def sban(self) -> bool:
             bool: whether the user was present in the banned list before the sban or not
         """
         if self.is_banned:
-            DbManager.delete_from(table_name="banned_users", where="user_id = %s", where_args=(self.user_id, ))
+            DbManager.delete_from(table_name="banned_users", where="user_id = %s", where_args=(self.user_id,))
             return True
         return False
```

Comment:
self.is_banned would rerun a db query to check if the user is banned, witch would always output false, since the user has just been sbanned or wasn't banned to begin with.
An approach would be to store the result of self.is_banned and then return it, like shown below, but i feel like this is simpler

```py
is_banned = self.is_banned
if is_banned:
    DbManager.delete_from(table_name="banned_users", where="user_id = %s", where_args=(self.user_id,))
return is_banned
```